pytradfri/device/__init__.py

```diff
@@ -1,7 +1,13 @@
 """Classes to interact with devices."""
 from datetime import datetime
 
-from pytradfri.const import (
+from pytradfri.device.blind_control import BlindControl
```

Comment:
Side note: It would be clearer to have all the names of pytradfri imported relatively to keep them grouped.